test/Core.Test/Services/CipherServiceTests.cs

```diff
@@ -1,65 +1,122 @@
 using System;
+using System.Threading.Tasks;
 using Bit.Core.Repositories;
 using Bit.Core.Services;
 using NSubstitute;
 using Xunit;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Table;
+using Core.Models.Data;
+using Bit.Core.Test.AutoFixture.CipherFixtures;
+using System.Collections.Generic;
+using AutoFixture;
+using Bit.Core.Test.AutoFixture;
 
 namespace Bit.Core.Test.Services
 {
     public class CipherServiceTests
     {
-        private readonly CipherService _sut;
+        [Theory, UserCipherAutoData]
+        public async Task SaveAsync_WrongRevisionDate_Throws(Cipher cipher)
+        {
+            var lastKnownRevisionDate = cipher.RevisionDate.AddDays(-1);
+            var sut = new Fixture().WithAutoNSubstitutions()
+                                   .For<CipherService>()
```

Comment:
These would get formatted as single-indent under the prior line (vs. inline with the dot above them)